source/extensions/filters/http/ext_proc/config.cc

```diff
@@ -19,8 +19,8 @@ Http::FilterFactoryCb ExternalProcessingFilterConfig::createFilterFactoryFromPro
   const auto filter_config = std::make_shared<FilterConfig>(
       proto_config, std::chrono::milliseconds(message_timeout_ms), max_message_timeout_ms,
       context.scope(), stats_prefix,
-      Envoy::Extensions::Filters::Common::Expr::getBuilder(context.serverFactoryContext()),
-      context.serverFactoryContext().localInfo());
+      Envoy::Extensions::Filters::Common::Expr::getBuilder(context.getServerFactoryContext()),
```

Comment:
I'm assuming this function was renamed upstream -- is that correct?

Reply:
yeah the `getServerFactoryContext` is the old function for this, which was changed [here](https://github.com/envoyproxy/envoy/pull/31150/files#diff-6eb911fbc2179b0e242b66ffb4d97d814b014d348c3f0c70d80d8db039e6e779), which should get picked up in 1.29